lib/index-finance.js

```diff
@@ -1,17 +1,27 @@
 'use strict';
 
-var Plotly = require('./core');
+var core = require('./core');
 
-Plotly.register([
+core.register([
+    // traces
     require('./bar'),
     require('./histogram'),
+    require('./funnel'),
+    require('./waterfall'),
     require('./pie'),
     require('./funnelarea'),
+    require('./indicator'),
     require('./ohlc'),
     require('./candlestick'),
-    require('./funnel'),
-    require('./waterfall'),
-    require('./indicator')
```

Comment:
As you can see, `funnel` and `waterfall` and `indicator` used to be registered in a different order in `finance` bundles.